lms/djangoapps/mobile_api/users/serializers.py

```diff
@@ -50,11 +53,7 @@ def to_representation(self, course_overview):
                 }
             },
             'course_image': course_overview.course_image_url,
-            'course_about': reverse(
-                'about_course',
-                kwargs={'course_id': course_id},
-                request=request,
-            ),
+            'course_about': None if not course_about_url else "http:{}".format(course_about_url),
```

Comment:
According to @ahsan-ul-haq:
"About page is in marketing site and lms is separate and we are not sure on which protocol either of them is running so to be on safe side we are striping it to be consistent"